src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/MixedMode_x64_VS2015_Default.exe.sarif

```diff
@@ -1171,7 +1171,7 @@
                   "text": "'{0}' was compiled with LinkTimeCodeGeneration (/LTCG) enabled."
                 },
                 "Warning": {
-                  "text": "'{0}' was compiled with without Link Time Code Generation (/LTCG). Enabling LTCG can improve optimizations and performance."
+                  "text": "'{0}' was compiled without Link Time Code Generation (/LTCG). Enabling LTCG can improve optimizations and performance."
```

Comment:
>compiled [](http://example.com/codeflow?start=37&length=8)

this edit came from a previous fix where we didn't rebaseline. the functional tests don't break currently on rule metadata changes.